code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -1218,28 +1218,19 @@
 	var/monkey_type = "Monkey"
 	list_reagents = list("nutriment" = 2)
 
-/obj/item/weapon/reagent_containers/food/snacks/monkeycube/afterattack(obj/O, mob/user, proximity)
-	if(!proximity)
-		return
-	if(istype(O, /obj/structure/sink))
-		to_chat(user, "<span class='notice'>You place [src] under a stream of water...</span>")
-		user.drop_item()
-		forceMove(get_turf(O))
-		return Expand()
-	..()
-
 /obj/item/weapon/reagent_containers/food/snacks/monkeycube/water_act(volume, temperature)
 	if(volume >= 5)
 		return Expand()
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wash(mob/user, atom/source)
-	if(do_after(user, 40, target = source))
-		return 1
+	user.drop_item()
+	forceMove(get_turf(source))
+	return 1
```

Comment:
Finally these damn things GC.

This looks like a change, but it's not; the intention for a monkeycube was that you would click it on the sink, where upon it would instantly convert to a moneky. What was actually happening is you'd use the monkey cube on the sink, there would be a delay, then it would make a monkey. `afterattack` never ran, as the item was `qdel`'d at that point.

Since `wash` wasn't calling `user.drop_item`, there was still a reference on the mob for the cube, causing it to fail GCing.